### ISSUE TITLE
Data Model Administration: Focus Traversal Keyboard Navigation Using Tab

### ISSUE THREAD

**Issue by enqueue** [metabase/metabase#12807]
**Describe the bug**
Some fields in the data model administration screen are not reachable using regular keyboard navigation using tab.

**Logs**
* No JavaScript errors or warnings in Firefox developer tools

**To Reproduce**
1. Go to 'Metabase Admin / Data Model'
2. Select a table with more than one queryable field
3. Set focus to the "column" field in the first row in the "columns" section
4. Hit tab twice

**Expected behavior**
Input focus on field "visibility"

**Actual behavior**
Input focus on gear / cog field

**Screenshots**

![metabase_datamodel_focustraversal](https://user-images.githubusercontent.com/5407006/85986160-168d4980-b9ec-11ea-88bd-ba17372bdd8d.gif)

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Browser: Mozilla/5.0 (X11; Linux x86_64; rv:77.0) Gecko/20100101 Firefox/77.0
- Operating System: x86_64 GNU/Linux
- Your databases: ClickHouse
- Metabase version: 0.35.4
- Metabase hosting environment: RHEL 6
- Metabase internal database: MySQL

**Severity**

This bug makes changes to larger data models a little bit more difficult.

**Comment by flamber** [comment]
Related to #1462 and #5686 - it's not specific to Data Model, but the Select component in general.
